pom.xml

```diff
@@ -79,8 +79,8 @@ THE SOFTWARE.
     <dependency>
       <groupId>org.jenkins-ci.main</groupId>
       <artifactId>jenkins-war</artifactId>
-      <!--to have access to User.getById-->
-      <version>1.651.2</version>
+      <!--to run with plugin unbundling by default-->
+      <version>2.7.3</version>
```

Comment:
Then you would need to do some cleanup. `TestPluginManager` has distinct modes for 1.x vs. 2.x.

Reply:
…which I guess did not get cleaned up in #135, so maybe do that here?

Reply:
I believe we will wipe 1.x entirely after https://groups.google.com/forum/#!topic/jenkinsci-dev/SdMOMKs7XIQ